pjlib/include/pj/log.h

```diff
@@ -95,8 +95,12 @@ enum pj_log_decoration
   static int trc_id = 0;                                                      \
   pj_log_ram_trace *pj_log_ramtrace_trace = pj_log_get_ram_trace_func();      \
   pj_log_ram_cache *pj_log_ramtrace_cache = pj_log_get_ram_cache_func();      \
-  (*pj_log_ramtrace_cache)(&trc_id,__FILE__,__LINE__,__VA_ARGS__);            \
-  (*pj_log_ramtrace_trace)(trc_id,__VA_ARGS__);                               \
+  if (pj_log_ramtrace_cache != NULL)                                          \
+  {                                                                           \
+    pj_assert(pj_log_ramtrace_trace != NULL);                                 \
```

Comment:
is there any way we can actually hit this line, given the above if statement? feels like the assert isn't doing anything here